hl7/4.2/modules/ROOT/pages/hl7-connector-reference.adoc

```diff
@@ -47,80 +69,118 @@
-| Fail when value length outside allowed range a| Boolean |  Fail when receive value length outside allowed range flag. If true, a message with this error is rejected; if false, the value is used anyway and the message is not rejected. In either case the error is logged and reported in an ERR segment. |  false |
-| Fail when invalid character in value a| Boolean |  Fail when receive value contains invalid characters flag. If true, a message with this error is rejected; if false, the character is either passed through or substituted and the message is not rejected. In either case the error is reported in an ERR segment. |  false |
-| Fail when too many repeats of value a| Boolean |  Fail when receive value is repeated too many times. If true, a message with this error is rejected; if false, the value is accepted and the message is not rejected. In either case the error is reported in an ERR segment. |  false |
-| Fail when unknown segment in message a| Boolean |  Fail when an unknown segment is present in a message. If true, a message with this error is rejected; if false, the segment is ignored and the message is not rejected. In either case the error is reported in an ERR segment. |  false |
-| Fail when segment out of order in message set a| Boolean |  Fail when a segment is out of order in a message set. If true, a message with this error is rejected; if false and the segment can be reordered the message is not rejected. In either case the error is reported in an ERR segment. |  false |
-| Fail when unused segment included in message set a| Boolean |  Fail when a segment marked as Unused is included in a message set. If true, a message with this error is rejected; if false, the message is not rejected and the unused segment is ignored. In either case the error is reported in an ERR segment. |  false |
-| Fail when too many repeats of segment a| Boolean |  Fail when a segment occurs too many times in a message set. If true, a message with this error is rejected; if false, the message is not rejected. In either case the error is reported in an ERR segment. |  false |
+** TRAINING |  Value to match the MSH-11-1 processing ID. If set, only messages with the specified processing ID are accepted for processing. Messages with other processing ID values or with no processing ID specified are rejected with an exception. |  |
+| Pattern for generic extension segment names a| String |  Java regular pattern for segment names to treat as generic extension segments. If specified, segment names matching this pattern are allowed anywhere in the message and are converted to maps of string values for elements and composites. |  |
+| Fail when required value missing a| Boolean a| Whether to reject a message if a required value is missing: 
+
+* If `true`, a message with this error is rejected.
+* If `false`, the value is ignored and the message is accepted. 
+
+In either case, the error is logged and reported in an ERR segment. |  false |
```

Comment:
Not super important but global: make false -> `false`

Reply:
This is interesting. I don't use backticks for values in the default column. I interpret the use of backticks as useful when something is in context to distinguish the value from the other words. We should discuss this.